server/real_estate/src/main/java/com/realestate/real_estate/services/BrokerService.java

```diff
@@ -14,23 +16,23 @@ public class BrokerService {
     @Autowired
     private PropertyRepository propertyRepository;
 
-    public List<Property> getAllProperties() {
+    public List<PropertyDetails> getAllProperties() {
```

Comment:
One of my endpoint depends on this method. Please don't change it. If you need another method you can create a new one.

Reply:
Fixed it